packages/timeline-state-resolver/src/__mocks__/casparcg-connection.ts

```diff
@@ -1,22 +1,16 @@
 // mock CasparCG
-import {
-	AMCP as orgAMCP,
-	AMCPUtil as orgAMCPUtil,
-	Command as orgCommand,
-	CasparCGSocketStatusEvent as orgCasparCGSocketStatusEvent,
-} from 'casparcg-connection'
+import { Commands as orgCommands, AMCPCommand as orgAMCPCommand, SendResult } from 'casparcg-connection'
+import { ResponseTypes } from 'casparcg-connection/dist/connection'
 import { EventEmitter } from 'events'
 
 const mockDo = jest.fn()
 
-const instances: Array<CasparCG> = []
+const instances: Array<BasicCasparCGAPI> = []
 
-export const AMCP = orgAMCP
-export const AMCPUtil = orgAMCPUtil
-export const Command = orgCommand
-export const CasparCGSocketStatusEvent = orgCasparCGSocketStatusEvent
+export const Commands = orgCommands
+export type AMCPCommand = orgAMCPCommand
 
-export class CasparCG extends EventEmitter {
+export class BasicCasparCGAPI extends EventEmitter {
```

Comment:
To make the two different acronyms clearly stand out it would be nice to format it `BasicCasparCgApi`.